Library/Formula/iojs.rb

```diff
@@ -0,0 +1,125 @@
+class Iojs < Formula
+  homepage "https://iojs.org/"
+  url "https://iojs.org/dist/v1.0.3/iojs-v1.0.3.tar.gz"
+  sha256 "c7fe7f71d9920f4cfd930f9022c7dddca7e11a0377fed1ee23e3241c2952db42"
+
+  option "with-debug", "Build with debugger hooks"
+  option "with-completion", "install bash completion for npm"
```

Comment:
I didn't know if that was too clashing against Node, given this deliberately removes the conflict and keg mechanism? *(As an experiment. Not saying this is the best way to get it done, just an alternative to the other discussion)*. I suppose I could do a little if file already exists don't replace it detection dance?